include/tester/ssp/SoftphoneManager.hpp

```diff
@@ -42,31 +42,31 @@ class SoftphoneManager
         }
         catch(pj::Error & err)
         {
-            std::cout << "Exception: " << err.info() << std::endl;
+            std::cerr << "Exception: " << err.info() << std::endl;
         }
     }
 
     void createSoftphone(int number_of_softphones)
     {
-        SoftphoneArguments sp_a;
-        sp_a.domain = _PBX_IP;
-        sp_a.secret = "12345678";
-        sp_a.timeout = 5000;
-        //sp_a.id = 1000;
+        SoftphoneArguments args;
+        args.domain = _domain;
+        args.secret = "12345678";
+        args.timeout = 5000;
+        //args.id = 1000;
 
         for(int i = 0; i < (number_of_softphones * 2); i++)
         {
-            sp_a.id = std::to_string(i + 1000);
-            Softphone sp(sp_a);
-            _softphone_map.insert({sp_a.id, &sp});
+            args.id = std::to_string(i + 1000);
+            Softphone sp(args);
+            _softphone_map.insert({args.id, sp});
```

Comment:
you're passing sp by value, so either use `std::move` OR use the constructor explicitly (without a variable)